Fix students context menu

mikeyarce opened the pull request:
### Changes proposed in this Pull Request

* Update `@wp-components` to version `wp-6.1`

With the old component, there is a problem with the students context menu not showing properly.  Updating to the latest component fixes that issue.

### Testing instructions

* Go to Sensei -> Students
* Click the `...` menu
* Make sure it opens up

Example:  
![student-context-menu](https://user-images.githubusercontent.com/3220162/202825245-cf52fa13-d90f-4195-b34c-672580357d88.gif)


Also, I wasn't sure if I needed to push `package-lock.json`?   It wasn't in `.gitignore` so.... let me know!

mikeyarce committed:
Update wordpress/omponents to wp-6.1 version

onubrooks reviewed (commented):
Fair enough. Some JS tests seem to be failing though so I would check and fix them first.

mikeyarce committed:
Merge branch 'trunk' into fix/students-context-menu

mikeyarce committed:
Fix student action menu test

Updating the wp-components package required updates to the tests so that we're following the right methods.

gikaragia commented:
Hey @mikeyarce, I think we need to resolve the conflicts on this one.

mikeyarce commented:
It was easier to just push a new branch so closing this one in favour of: https://github.com/Automattic/sensei/pull/6440
